src/app/components/team/TeamTags.js

```diff
@@ -307,95 +308,100 @@ class TeamTagsComponent extends Component {
 
   tagsList(list) {
     return (
-      <List
-        style={{
-          maxHeight: 400,
-          overflowY: 'auto',
-          overflowX: 'hidden',
-          padding: units(2),
```

Comment:
Please remove units from import statement.

`27:10  error  'units' is defined but never used  no-unused-vars`